packages/wallet-apis/src/provider.ts

```diff
@@ -193,6 +194,12 @@ export const createEip1193ProviderFromClient = <
           })(params);
         }
 
+        case "wallet_getCapabilities": {
+          return handler<"wallet_getCapabilities">(async () => {
+            return await getCapabilities(client);
+          })(params);
+        }
```

Comment:
do we need this here? for `wallet_getCallsStatus`, we just let the fallback below handle if, since no transformations are needed.

if you prefer to be explicit like this though, it might be good to add`wallet_getCallsStatus` here too?

Reply:
Ah I see, I added this here mostly because the catchall clause still had a todo comment that made it unclear if we would keep it.

I'm leaning towards keeping it and adding `wallet_getCallsStatus`, mostly just as a reader benefit to make it obvious that we support it. Wdyt?

Reply:
yeah, i think that makes sense.

Reply:
Hmm, having some typing issues with `wallet_getCallsStatus`, it seems like the corresponding viem action annotates the return type with some extra fields and also converts types a bit. I will leave it out for now.